roles/dnsmasq/templates/dnsmasq-rc.yml

```diff
@@ -1,16 +1,21 @@
 ---
-apiVersion: extensions/v1beta1
-kind: DaemonSet
+apiVersion: v1
+kind: ReplicationController
```

Comment:
Then I would have to make a separate name for dnsmasq-deploy.yml (probably dnsmasq-autoscaler.yml). Any particular benefit except rc is an oldschool resource type?